src/main/kotlin/minesweeper/entity/MineField.kt

```diff
@@ -20,7 +20,52 @@ class MineField(
 
     fun countAroundMines(coordinate: Coordinate): Int {
         return coordinate.adjacentCoordinates()
-            .filter { it.isWithinBounds(width.value, height.value) }
+            .filter { it.isWithinBounds(width, height) }
             .count { _cells.findCell(it) is Cell.Mine }
     }
+
+    fun open(coordinate: Coordinate) {
+        val cell = findCell(coordinate)
+        if (cell.isRevealed) return
+
+        cell.open()
+        if (shouldOpenAdjacentCells(cell, coordinate)) {
```

Comment:
생각해보니 어짜피 coordinate는 cell.coordinate와 동일할 것 같은데, shouldOpenAdjacentCells 메서드의 파라미터로 cell만 받아도 충분하지 않을까요? 
```suggestion
        if (shouldOpenAdjacentCells(cell, coordinate)) {
```

Reply:
오! 아! 그렇네요! 감사합니다. 👍